code/modules/mob/mob.dm

```diff
@@ -181,7 +181,7 @@
 	to_chat(src, msg, avoid_highlighting = avoid_highlighting)
 
 
-/atom/proc/visible_message(message, self_message, blind_message, vision_distance = DEFAULT_MESSAGE_RANGE, list/ignored_mobs, list/visible_message_flags, allow_inside_usr = FALSE)
+/atom/proc/visible_message(message, self_message, blind_message, vision_distance = DEFAULT_MESSAGE_RANGE, list/ignored_mobs, list/visible_message_flags, allow_inside_usr = FALSE, separation = " ")
```

Comment:
I am unsure if the default should be space. Doesn't this impact other cases than emotes etc?
Also this proc should handle null separator

Reply:
from the looks of the code, the seperation is just the character that comes between the name and the message. The default was space before.
The only situation where you would want this as something like null, would be '*me 's suit sensors flicker out.'